### DISCUSSION TITLE
Boy! Did things get busy here…

### DISCUSSION THREAD

**Discussion by kalebmckale** [pypa/pipenv#5830] [General]
@matteius After posting my comments and scrolling through other issues, I have to say, "Wow!" I look away for what seemed a short time and the number of issues exploded. I hope you're taking care of yourself and not getting overwhelmed. Still working on getter a better-than-iPad solution for coding at home so I can make more significant contributions.

**Comment by matteius** [comment]
I think we are on track, if I look back to https://github.com/pypa/pipenv/issues/5131
We were targeting ~<200 issues by summer of this year and we are currently at 191.   When  the draft-no-reqslib branch gets released, a number of other issues will be closed and possibly some new ones opened, but so far its fairly normal.   I do think dropping requirementslib will open the door to more stable code base, and additionally some improvements have been requested in pythonfinder that if implemented would further reduce the issue count and even allow us to drop pydantic!

> Still working on getter a better-than-iPad solution for coding at home so I can make more significant contributions.

Awesome, that is great to hear!!
